src/Text.cpp

```diff
@@ -0,0 +1,40 @@
+#include <Text.hpp>
+
+#include <Asset.hpp>
+#include <Logger.hpp>
+
+#include <ft2build.h>
+#include FT_FREETYPE_H
+
+#include <filesystem>
+#include <stdexcept>
+
+using namespace resources;
+
+struct Font::Data {
+  FT_Library ft;
+  FT_Face face;
+};
+
+Font::Font(std::string fontName, uint8_t fontSize)
+    : fontData{std::make_unique<Font::Data>()} {
+  if (FT_Init_FreeType(&fontData->ft)) {
+    logging::Logger::LogError("FREETYPE: Could not init FreeType Library");
+    throw new std::runtime_error{"FREETYPE: Could not init FreeType Library"};
+  }
+
+  // Windows uses a wchar_t which we don't want to use because we don't
+  // support internationalization.

```

Comment:
Not today. The problem is that internationalization would require us to change everything to wchar_t and std::wstring and I guess while it would be simple, we are the only people looking at the text output. Also, I think if we did internationalization it would be when we load text to the screen.